nucliadb_node/binding/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # Changelog
 
+## 0.5.10
```

Comment:
Isn't that a new feature? Why are you incrementing the third number instead of the second?

Reply:
👍🏼 I thought we didn't really care about semantic versioning at this point